<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smallMetexLab</class>
 <widget class="QMainWindow" name="smallMetexLab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Small Metex Lab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="digitalDisplayBox">
         <property name="title">
          <string>Digital display</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QLabel" name="funcionLabel">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Voltage</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="acdcLabel">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>DC</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="3">
           <widget class="QLCDNumber" name="lcdNumber">
            <property name="digitCount">
             <number>4</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="unitLable">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>mV</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="analogDisplayBox">
         <property name="title">
          <string>Analog display</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="plotBox">
         <property name="title">
          <string>Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QGroupBox" name="displayModBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="title">
          <string>Display mode</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0" colspan="3">
           <widget class="QRadioButton" name="analogRadioButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Analog</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="xAxisPlotLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X-axis :</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="plotRadioButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Plo&amp;t</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="xAxisPlotComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Measurement number</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="digitalRadioButton">
            <property name="text">
             <string>Di&amp;gital</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="modeBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="title">
          <string>Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="autoRadioButton">
            <property name="text">
             <string>Au&amp;to</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="manualRadioButton">
            <property name="text">
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="transistorTestRadioutton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Trans&amp;istor test</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="measureBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="title">
          <string>Measure</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QSpinBox" name="intervalSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>200</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="intervalLabel">
            <property name="text">
             <string>Interval [ms]:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QPushButton" name="manualButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Take a measure</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="logBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>130</height>
          </size>
         </property>
         <property name="title">
          <string>Log to file</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QComboBox" name="separatorComboBox">
            <item>
             <property name="text">
              <string>;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tab</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Space</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="xAxisLabel">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X-axis:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="5">
           <widget class="QLineEdit" name="fileLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="separatorLabel">
            <property name="text">
             <string>Sparator:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="filePushButton">
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Path to file:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="xAxisComboBox">
            <item>
             <property name="text">
              <string>Time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Measurement number</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="5" colspan="2">
           <widget class="QPushButton" name="logPushButton">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Log on</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>140</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="connectBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="title">
          <string>Connection</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QComboBox" name="portBox"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="connectButton">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="portNameLabel">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Port name:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionZamknij">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>C&amp;onfiguration</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>

